Marlin/src/module/ft_motion.cpp

```diff
@@ -449,7 +459,7 @@ void FTMotion::reset() {
   endPosn_prevBlock.reset();
 
   makeVector_idx = makeVector_idx_z1 = 0;
-  makeVector_batchIdx = 0;
+  makeVector_batchIdx = TERN(FTM_UNIFIED_BWS, 0, _MAX(last_batchIdx, FTM_BATCH_SIZE));
```

Comment:
If `FTM_WINDOW_SIZE` must always be a multiple of `FTM_BATCH_SIZE` then we can enforce that in a sanity check, and this can simply be `makeVector_batchIdx = last_batchIdx`.

Reply:
… The same applies to lines 153-159.

Reply:
The trick is to keep `max_intervals` as a  `FTM_BATCH_SIZE` multiple.